### ISSUE TITLE
Migrate obermann articles

### ISSUE THREAD

**Issue by richardbporter** [uiowa/uiowa#3697]
> For instance, our current website has lots of news articles (using the “article” content type). We’d love if those could be moved over to SiteNow as individual articles. I know we’d have to manually tag them if we wanted to filter and display them, but the more important thing is that they all get moved over, so that they’re accessible and searchable on the new site. Each of our current article nodes use the Title field, the featured image, the body, and one or more “Featured Program” tags, along with the date of publication. The SiteNow article content type has those same fields—so maybe all of that content could be moved over (except for maybe the tagged programs)?

**Comment by briand44** [comment]
> Let’s hold off on the news migrations until we’re close to launch. I don’t have a date for that yet, but I’m guessing it will be mid- to late fall (early November, maybe). We would definitely like you to apply the tags to the articles. I have already created those tags (duplicates of the tags we use on our current Drupal site).
 
>I will manually migrate the most recent articles myself, so that your team can automatically migrate all articles created between X dates. (Again, I’ll let you know the exact dates when we’re closer to launch). The reason I’d like to manually migrate the most recent articles is that we’ve decided not to use SiteNow’s featured image field for articles, since the images we select don’t function well as large rectangular banners. So I’ve been manually inserting the featured images into the articles’ text fields. For the older articles that you’ll be auto-migrating, using the Featured Image field is OK with us. We just want the most recent news articles to look more polished, if that makes sense.

**Comment by briand44** [comment]
- Migrate all articles created 9/9/2020 and prior (They manually moved over more recent ones).
- Map tags from old site (These do not need created, just mapped, as they manually created these).
- create redirects from old articles
- provide list of articles with internal links that will likely be broken

**Comment by briand44** [comment]
They said site is almost ready for launch but they are waiting on this migration.

**Comment by bspeare** [comment]
@briand44 On the D7 site they are using the "tags" field and also have a custom entity reference to a content type called "Featured Program", do you know whether they want one or the other migrated?

**Comment by briand44** [comment]
I know they wanted to map the tags, they have already created them so those just need mapped not recreated. I'm checking on the programs... cc'ing you.
